docs/versioned-plugins/integrations/aws-v7.0.0.asciidoc

```diff
@@ -17,13 +17,13 @@ END - GENERATED VARIABLES, DO NOT EDIT!
 
 === AWS Integration Plugin {version}
 
-// include::{include_path}/plugin_header.asciidoc[]
+include::{include_path}/plugin_header.asciidoc[]
 
 ==== Description
 
 The AWS Integration Plugin provides integrated plugins for working with Amazon Web Services:
 
-////
+
  - {logstash-ref}/plugins-codecs-cloudfront.html[Cloudfront Codec Plugin]
```

Comment:
Note that the links in this list of components point to the Logstash Reference.  They will continue to point to the (old) stand-alone versions of these plugins until I finish all necessary updates to the various stack branches.